metadata-ingestion/src/datahub/ingestion/source/feast.py

```diff
@@ -81,6 +88,14 @@
     feast.types.Invalid: MLFeatureDataType.UNKNOWN,
 }
 
+# FIXME: Update to have more owners
+_owner_mapping: Dict[str, Dict[str, Any]] = {
+    "Datahub": {
+        "owner_type": builder.OwnerType.USER,
```

Comment:
Does feast support GROUP owners as well? Thanks for calling this out!

Reply:
yes, we can pass the owner as a "Person" (user) or "Group"
<img width="248" alt="Screenshot 2024-11-08 at 17 00 44" src="https://github.com/user-attachments/assets/51a24557-8bbd-4e52-b56e-ff0af16ca983">

Reply:
the issue is that from Feast side I can only specify the owner: str (so, for example "Margarida") and this should map to a OwnerType.USER and to the appropriate OwnershipTypeClass.

Reply:
ok got it !